testing/k3s/APKBUILD

```diff
@@ -0,0 +1,67 @@
+# Contributor: Oleg Titov <oleg.titov@gmail.com>
+# Maintainer: Oleg Titov <oleg.titov@gmail.com>
+pkgname=k3s
+pkgver=0.4.0
+pkgrel=0
+pkgdesc="Lightweigt Kubernetes. 5 less than k8s."
+url="https://k3s.io"
+arch="all"
+license="Apache-2.0"
+depends="containerd"
+makedepends="go linux-headers"
+install=""
+subpackages="$pkgname-doc"
+source="$pkgname-$pkgver.tar.gz::https://github.com/rancher/k3s/archive/v$pkgver.tar.gz"
+builddir="$srcdir/src/github.com/rancher/$pkgname"
+
+prepare() {
+	export GOPATH="$srcdir"
+
+	mkdir -p $(dirname $builddir)
+	mv "$srcdir"/$pkgname-$pkgver "$builddir"/
+
+	default_prepare
+}
+
+build() {
+	msg2 "Prepare GOPATH and GOBIN..."
+	export GOPATH="$srcdir"
+	export GOBIN="$GOPATH/bin"
+
+	msg2 "Building k3s..."
```

Comment:
not sure it makes sense to keep it

Reply:
There is no any output during build process of each of 3 targets and the build process takes some time. I used these msg2 to have a visual indication of build progress and this gives me an idea which target produces errors if any. That's the motto behind use of msg2. If you feel that removing them would be beneficial it would be my pleasure to incorporate these changes.